src/account/AccountList.js

```diff
@@ -18,16 +18,20 @@ export default class AccountList extends React.PureComponent {
 
     return (
       <View>
-        {accounts.map((account, i) =>
-          <AccountItem
-            key={account.apiKey}
-            index={i}
-            showDoneIcon={i === 0 && auth.apiKey !== '' && auth.apiKey === account.apiKey}
-            {...account}
-            onSelect={onAccountSelect}
-            onRemove={onAccountRemove}
-          />,
-        )}
+        <FlatList
+          keyboardShouldPersistTaps="always"
+          data={accounts}
+          keyExtractor={item => `${item.email}${item.realm}`}
```

Comment:
Can't use apiKey as there may be cases where two accounts have empty apiKey (logged out ones). Can't use email as one email can be registered on different realms. Can't use realm as there may be multiple accounts with same realm. So combining email and realm.